cpp/tests/test_shuffler.cpp

```diff
@@ -355,6 +356,8 @@ class ConcurrentShuffleTest
             stream,
             br.get()
         ));
+
+        shuffler.shutdown();
```

Comment:
Not sure I follow, why did `shutdown()` was removed from `test_shuffler` to be added here?

Reply:
I understand. This was just to explicitly indicate that shuffler is shutdown. It's more verbose and easy to follow. 

But its fine. I'll remove this line.